recipes/mold/1.4.x/test_package/conanfile.py

```diff
@@ -0,0 +1,23 @@
+import os
+from conan import ConanFile
+from conan.tools.build import cross_building
+from conan.tools.layout import basic_layout
+
+
+class MoldTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    # VirtualRunEnv can be avoided if "tools.env.virtualenv:auto_use" is defined
+    # (it will be defined in Conan 2.0)
+    generators = "VirtualRunEnv"
+    apply_env = False
```

Comment:
Why do you need apply_env False? Remember users should not need to set it to use that package.

Reply:
most probably copy-paste mistake